requirements.txt

```diff
@@ -1,7 +1,8 @@
 Django==4.2.4
-find-similar==1.0.1
+find-similar
```

Comment:
API can use different versions of library (not only last). In this way we can control API version independent of `find-similar` library version. Now we can use 1.2.0 version. Can you fix this concrete version in `requirements.txt`?

Reply:
Ok, i will set it to find-similar==1.2.0